spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/registry/filter/DefaultRegisterFilterHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.tencent.cloud.polaris.registry.filter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultRegisterFilterHandler implements RegisterFilterHandler {
+	private static final Logger log = LoggerFactory
+			.getLogger(DefaultRegisterFilterHandler.class);
+
+	@Override
+	public boolean beforeInvoke(RegisterFilterHandlerContext registerFilterHandlerContext) {
+		log.info("defaultRegisterFilterHandler beforeInvoke info");
```

Comment:
Maybe change log level to debug, or just delete DefaultRegisterFilterHandler. I think this is just used for test for the time being.